### PR TITLE
Fix ie and tests

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/layers/TrajservLayer.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import 'jest-canvas-mock';
+// import Map from 'ol/Map';
```
Comment (edellucien) [inline src/layers/TrajservLayer.test.js]:
```suggestion
```

Comment (edellucien) [reply src/layers/TrajservLayer.test.js]: Some comments to be removed in this test file.

Comment (oterral) [reply src/layers/TrajservLayer.test.js]: removed

**Comment by edellucien** [comment]
Perfect!

**Pull request merged**